.rubocop.yml

```diff
@@ -3,3 +3,6 @@ Metrics/LineLength:
 
 Metrics/BlockLength:
   Max: 100
+
+Style/SymbolArray:
+  Enabled: false
```

Comment:
@MWilsonPuppet, rather than disabling this check, do you just want to fix the Rakefile? e.g.

-    task publish: [
-      :push,
-      :update_microbadger
+    task publish: %i[
+      push
+      update_microbadger